pkg/resolver/search.go

```diff
@@ -77,62 +80,36 @@ func (s *SearchResult) Related() []SearchRelatedResult {
 	return r
 }
 
-//=====================
-
-var trimAND string = " AND "
-
 func (s *SearchResult) buildSearchQuery(ctx context.Context, count bool) (string, []interface{}) {
-	var selectClause, whereClause, limitClause, limitStr, query string
-	var args []interface{}
-	// SELECT uid, cluster, data FROM search.resources  WHERE lower(data->> 'kind') IN (lower('Pod')) AND lower(data->> 'cluster') IN (lower('local-cluster')) LIMIT 10000
-	selectClause = "SELECT uid, cluster, data FROM search.resources "
+	var limit int
+	var selectDs *goqu.SelectDataset
+	var whereDs []exp.Expression
+
+	// Example query: SELECT uid, cluster, data FROM search.resources  WHERE lower(data->> 'kind') IN
+	// (lower('Pod')) AND lower(data->> 'cluster') IN (lower('local-cluster')) LIMIT 10000
+	//FROM CLAUSE
+	schemaTable := goqu.S("search").Table("resources")
+	ds := goqu.From(schemaTable)
+	//SELECT CLAUSE
+	selectDs = ds.Select("uid", "cluster", "data")
```

Comment:
Move this to else to avoid reassigning.
```
if count {
	selectDs = ds.Select(goqu.COUNT("uid"))
} else {
	selectDs = ds.Select("uid", "cluster", "data")
}
```